tests/common/Touch.Unit/Touch.Server/Main.cs

```diff
@@ -0,0 +1,400 @@
+// Main.cs: Touch.Unit Simple Server
```

Comment:
We don't need any of the Touch.Server code.

Reply:
Removed